zebrad/tests/acceptance.rs

```diff
@@ -4,195 +4,292 @@
 #![warn(warnings, missing_docs, trivial_casts, unused_qualifications)]
 #![forbid(unsafe_code)]
 
-use abscissa_core::testing::prelude::*;
 use color_eyre::eyre::Result;
-use once_cell::sync::Lazy;
 use std::time::Duration;
 use zebra_test::prelude::*;
 
-/// Executes zebrad binary via `cargo run`.
-pub static RUNNER: Lazy<CmdRunner> = Lazy::new(CmdRunner::default);
+// Todo: The following 3 helper functions can probably be abstracted into one
+fn get_child_single_arg(arg: &str) -> Result<zebra_test::command::TestChild> {
+    let (mut cmd, _guard) = test_cmd(env!("CARGO_BIN_EXE_zebrad"))?;
```

Comment:
Its important to not drop this `_guard`, doing so will delete the temp dir that this command is going to run inside of.

Reply:
Thanks, fixed this at https://github.com/ZcashFoundation/zebra/pull/690/commits/5f038d540267a53f6ae4e80756e5af9a2818f5f6